app/controllers/chapters_controller.rb

```diff
@@ -161,6 +161,9 @@ def update
       posted_changed = @chapter.posted_changed?
       @work.set_revised_at_by_chapter(@chapter)
       if @chapter.save && @work.save
+        if @chapter.posted
```

Comment:
Because this is so short, it might make sense to do it all on one line, like: `post_chapter if @chapter.posted`, but this is also fine!